### PR TITLE
[#5] Renaming and Cleanup

### PR DESCRIPTION
**Pull request by ThomasObenaus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,181 +1,25 @@
-# gremtune
+# gremcos
 
-[![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)](http://godoc.org/github.com/schwartzmx/gremtune) [![Build Status](https://travis-ci.org/schwartzmx/gremtune.svg?branch=master)](https://travis-ci.org/schwartzmx/gremtune) [![Go Report Card](https://goreportcard.com/badge/github.com/schwartzmx/gremtune)](https://goreportcard.com/report/github.com/schwartzmx/gremtune)
+[![GoDoc](https://godoc.org/github.com/supplyon/gremcos?status.svg)](https://godoc.org/github.com/supplyon/gremcos) ![build](https://github.com/supplyon/gremcos/workflows/build/badge.svg?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/supplyon/gremcos)](https://goreportcard.com/report/github.com/supplyon/gremcos)
 
-- [ ] **Adjust README --> go-gremlin-cosmos**
+Gremcos is a fork of [schwartzmx/gremtune](https://github.com/schwartzmx/gremtune) with alterations to make it compatible with [Azure Cosmos](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) which is a Graph Database (Gremlin API) for Azure.
```
Comment (dArignac) [inline README.md]: we make it compatible to the "Gremlin API of Azure Cosmos DB". Cosmos offers more than just the graph part, that needs to be clear here.

File: README.md [hunk 2]
```diff
@@ -1,181 +1,25 @@
...
-
-**Modifications were made to `gremgo` in order to "support" AWS Neptune's lack of Gremlin-specific features, like no support query bindings among others. See differences in Gremlin support here: [AWS Neptune Gremlin Implementation Differences](https://docs.aws.amazon.com/neptune/latest/userguide/access-graph-gremlin-differences.html)**
+Gremcos is a fast, efficient, and easy-to-use client for the [TinkerPop](http://tinkerpop.apache.org/docs/current/reference/) graph database stack. It is a gremlin language driver which uses WebSockets to interface with [gremlin server](http://tinkerpop.apache.org/docs/current/reference/#gremlin-server) and has a strong emphasis on concurrency and scalability. Please keep in mind that gremcos is still under heavy development and although effort is being made to fully cover gremcos with reliable tests, bugs may be present in several areas.
```
Comment (dArignac) [inline README.md]: Should point out that we are tied to version 4.3.0 as that is the only working version we know of.

File: README.md [hunk 3]
```diff
@@ -1,181 +1,25 @@
...
-        return nil, err
-    }
-    fmt.Printf("%s", j)
-}
-```
+- See: [examples/README.md](examples/README.md)
 
 ## Hints
 
 This implementation supports [Graphson 2.0](http://tinkerpop.apache.org/docs/3.4.4/dev/io/#graphson-2d0) (not 3) in order to be compatible to CosmosDB. This means all the responses from the CosmosDB server as well as the responses from the local gremlin-server have to comply with the 2.0 format.
```
Comment (dArignac) [inline README.md]: 3.4.0 is only supported by us

File: README.md (hunk 2 above)
Comment (ThomasObenaus) [reply README.md]: added info in paragraph hints

File: README.md (hunk 3 above)
Comment (ThomasObenaus) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (ThomasObenaus) [reply README.md]: fixed

**Pull request merged**